R/manifest-convert.R

```diff
@@ -20,18 +21,11 @@
 #'   object; otherwise, the lockfile will be written to the path specified by
 #'   `lockfile`.
 #'
-#' @details
-#' By default the `lockfile` argument is set to `NA`. This will not create a new
-#' `renv.lock` file. Rather, it will return a lockfile object (see `?lockfile`)
-#' that can be used to create a new `renv.lock` file. If `lockfile` is set to a
-#' character string, a new file will be created with that path -- e.g.
-#' `renv.lock` -- and the lockfile object will be returned.
-#'
 #' @return
 #' An renv lockfile.
 #'
 #' @keywords internal
-renv_lockfile_from_manifest <- function(manifest,
+renv_lockfile_from_manifest <- function(manifest = "manifest.json",
```

Comment:
IMO this is going to be 90%+ of use cases, so might as well provide it as the default.

Should I also add an explicit check that this file exists in this function?